vocode/streaming/utils/create_task.py

```diff
@@ -6,25 +6,12 @@
 tasks_registry = []
 
 
-def log_if_exception(reraise_cancelled: bool, future: asyncio.Future) -> None:
-    try:
-        if exc := future.exception():
-            logger.exception(
-                f"Vocode wrapped logger; exception raised by task {future}: {exc}",
-                exc_info=exc,
-            )
-    except asyncio.CancelledError:
-        if reraise_cancelled:
-            raise
-
-
-def asyncio_create_task_with_done_error_log(
+def asyncio_create_task(
```

Comment:
this is fine - i understand why we need `tasks_registry`, but I wonder if we audit the use-cases and eventually remove this method

Reply:
The primary reason I removed this is because each exception log causes a duplicate error in sentry. And we can just catch them by the raised error. Is there a scenario where we actually need this method/functionality?

Reply:
i believe we keep tasks in `tasks_registry` so that these tasks don't get garbage collected when they aren't assigned to some state in a worker, e.g.

i was thinking we would eventually remove it and replace it with `asyncio.create_task`

Reply:
Oh sure, you're talking about the method in general, not the change I made.